torchaudio/functional.py

```diff
@@ -524,34 +524,38 @@ def lfilter(waveform, a_coeffs, b_coeffs):
 
     Returns:
         output_waveform (torch.Tensor): Dimension of `(n_channel, n_frames)`.  Output will be clipped to -1 to 1.
+                                        Will be on the same device as the inputs.
 
     """
 
     assert(waveform.dtype == torch.float32)
     assert(a_coeffs.size(0) == b_coeffs.size(0))
     assert(len(waveform.size()) == 2)
```

Comment:
Is that something mandatory?

Reply:
No, this is probably the easiest way.  We could also have the user specify where they want the output or have the user pass in an output container.

Reply:
We don't need to worry about this here. This can be as part of a separate PR.